scripts/simulator_i386.py

```
@@ -0,0 +1,10 @@
+Import("env")
```

Ruff (0.15.18)

[error] Undefined name `Import`

(F821)